assets/data/cards_pt-br.json

```
@@ -173,10 +202,12 @@
     {
       "title": "React Native",
       "description": "É um framework do Javascript baseado no React para desenvolvimento de aplicativos móveis multiplataforma."
```

Achei, faltou a vírgula aqui.
```suggestion
      "description": "É um framework do Javascript baseado no React para desenvolvimento de aplicativos móveis multiplataforma.",
```